lesson_3/readme.md

```diff
@@ -1,2 +1,42 @@
-# Order:
-1. `word_chain.md`
+# Django Girls Code Camp Week 3
+
+Requirements:
+
+- Finish CodeCademy
```

Comment:
How can we know their status of CodeAcademy ? Have we asked them to show us completion percentage ?